src/audio_call.c

```diff
@@ -63,36 +64,12 @@ extern ToxWindow *windows[MAX_WINDOWS_NUM];
 
 struct CallControl CallControl;
 
+extern struct Winthread Winthread;
+
 #define cbend pthread_exit(NULL)
 
 #define frame_size (CallControl.audio_sample_rate * CallControl.audio_frame_duration / 1000)
```

Comment:
frame_size is also never used. It's also a super confusing macro in and of itself, especially given that the name "frame_size" does appear quite often, but isn't this macro.